src/content/guides/typescript.mdx

```diff
@@ -94,6 +95,45 @@ module.exports = {
 
 This will direct webpack to _enter_ through `./index.ts`, _load_ all `.ts` and `.tsx` files through the `ts-loader`, and _output_ a `bundle.js` file in our current directory.
 
+Regarding webpack configuration, instead of using Javascript for `webpack.config.js`, we could use the Typescript version `webpack.config.ts`. This is possible by installing `ts-node`.
+
+```bash
```

Comment:
probably can have a link somewhere to link to https://webpack.js.org/configuration/configuration-languages/#typescript

which explains on using typescript for configuring webpack